src/gui/widgets/scroll_text.cpp

```diff
@@ -261,12 +269,14 @@ std::unique_ptr<widget> builder_scroll_text::build() const
 
 	widget->set_vertical_scrollbar_mode(vertical_scrollbar_mode);
 	widget->set_horizontal_scrollbar_mode(horizontal_scrollbar_mode);
-	widget->set_editable(editable);
-	widget->set_text_alignment(text_alignment);
 
 	const auto conf = widget->cast_config_to<scroll_text_definition>();
 	assert(conf);
 
+	widget->set_editable(editable);
```

Comment:
These are redundant. You can just set them from the `builder` object in the ctor.

Reply:
done, setting in the constructor now.